src/wp-admin/includes/update-core.php

```diff
@@ -7,6 +7,21 @@
  * @since 2.7.0
  */
 
+/**
+ * Preload files from the current version into memory that may be
+ * needed during the update process, and may not be available
+ * after files from the updated version have been installed.
+ *
+ * Why? Requests 2.0.0 (introduced in WordPress 5.9.0) introduced
+ * namespaces. If a class is needed after its file is upgraded but
+ * the class is not already available in memory, a fatal error is
+ * thrown because the fully-qualified name of the needed class
+ * is no longer available in the filesystem.
```

Comment:
```suggestion
```

Removing the Requests explanation as preloading files into memory may not be limited to Requests (now and/or in the future). Having this specific comment _may_ limit focus on the intent of identifying and preloading files that may be needed before those files are upgraded.